packages/components/date-picker/src/date-picker-com/panel-date-pick.vue

```diff
@@ -169,25 +169,27 @@
       </div>
     </div>
     <div v-show="footerVisible" :class="ppNs.e('footer')">
-      <el-button
-        v-show="!isMultipleType"
-        text
-        size="small"
-        :class="ppNs.e('link-btn')"
-        :disabled="disabledNow"
-        @click="changeToNow"
-      >
-        {{ t('el.datepicker.now') }}
-      </el-button>
-      <el-button
-        plain
-        size="small"
-        :class="ppNs.e('link-btn')"
-        :disabled="disabledConfirm"
-        @click="onConfirm"
-      >
-        {{ t('el.datepicker.confirm') }}
-      </el-button>
+      <slot name="footer" :confirm="onConfirm" :change-to-now="changeToNow">
```

Comment:
Perhaps `disabledNow` and `disabledConfirm` can also be passed to users through slots, making it easier for them to decide whether to add disabled styles.